src/components/Stories.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+export default function Stories({ trendingStories }) {
+  return (
+    <>
+      <div className="grid grid-cols-4 gap-4">
+        <div className="col-span-4 ...">
+          <div className="grid grid-cols-2 gap-4">
+            <div ClassName="...">
+              <h1 className="text-4xl text-left font-playfair font-bold">
+                Trending Stories
+              </h1>
+            </div>
+
+            <div ClassName="...">
+              <h1 className="text-right font-inter mx-1 font-semibold text-amber-500 py-6">
+                View all &gt;
+              </h1>
+            </div>
+          </div>
+        </div>
+
+        {trendingStories.map((img) => (
+          <div ClassName="...">
```

Comment:
Whenever mapping objects, make sure to use the `key` attribute. You can update this like so:
```
 {trendingStories.map((img, idx) => (
          <div className="image-container" key={idx}>
```

Reply:
updated